avaframe/in1Data/getInput.py

```diff
@@ -202,10 +203,10 @@ def getInputDataCom1DFAPy(avaDir, cfg, flagDev=False):
     if flagDev is True:
         releaseDir = 'devREL'
         relFiles = glob.glob(inputDir+os.sep + releaseDir+os.sep + '*.shp')
-    elif cfg['releaseScenario'] != '':
+    elif cfg['FLAGS']['releaseScenario'] != '':
```

Comment:
should this be in the flags? it is not a boolean..

Reply:
no this is if you want to specify a specific release. This was already in the ini flag section, I did not change this

Reply:
It was called like this before:
demFile, relFiles, entFiles, resFile, entResInfo = gI.getInputDataCom1DFAPy(
        avaDir, cfg['FLAGS'], flagDev)
    and now like this:
demFile, inputSimFiles = gI.getInputDataCom1DFAPy(avaDir, cfg, flagDev)